manifests/server/grant.pp

```diff
@@ -18,6 +18,7 @@
 # @param ensure Specifies whether to grant or revoke the privilege. Default is to grant the privilege. Valid values: 'present', 'absent'.
 # @param group Sets the OS group to run psql
 # @param psql_path Sets the path to psql command
+# @param instance The name of the Postgresql database instance. Default value: 'main'.
```

Comment:
we don't need to mention the default value, puppet-strings will pick it up automatically